tests/distributed/test_local_flow_local_remote_local/flow.yml

```diff
@@ -2,6 +2,7 @@
 version: '1'
 with:
   read_only: true
+  public-expose: true
```

Comment:
what is `public-expose`? seems irrelevant as the test passing anyway

Reply:
yes leftovers from previous hypothesis, my bad